controller-server/src/main/java/com/yahoo/vespa/hosted/controller/application/SystemApplication.java

```diff
@@ -68,13 +67,17 @@ public boolean configConvergedIn(ZoneId zone, Controller controller) {
         if (!hasApplicationPackage()) {
             return true;
         }
-        // TODO: Remove this hack once Docker hosts are removed from zone-application.
-        if (isAws(zone.region())) {
-            return true; // Skip checking config convergence on AWS as Docker hosts do not have cloud config
-        }
+        // TODO: Docker hosts running host admin cannot be checked. Since a zone can have
+        // Docker hosts running either host admin or node-admin, it's not possible to check
+        // config convergence, so we need to always return true here.
+        // We want to remove the line below and check config convergence for proxy nodes
+        // when all Docker hosts are running host admin
```

Comment:
Is the goal still to remove Docker hosts from zone-application?

Reply:
Yes, though we don't know exactly how we can accomplish that yet.

Reply:
Ok. I think it would be better if we could find a way to check convergence for proxy nodes only, as that's what we care the most about anyway.